pai-management/doc/how-to-setup-dev-box.md

```diff
@@ -1,11 +1,40 @@
 # Setup dev-box
 
-## Recommended: In docker Container
+## Install docker on your server
 
-- Install docker on your host.
-- Finish configuring the cluster-configuration
+```dev-box``` is a docker container you could use to maintain PAI cluster. And we provide a prebuild image hosted on Docker Hub for your convenience.
 
-#### Build the Dev-Box's docker image.
+## Use prebuild dev-box image
+
+```bash
+
+# Pull the dev-box image from Docker Hub
+sudo docker pull docker.io/openpai/dev-box
+
+# Run your dev-box
```

Comment:
please add a comment:

1. Assume the path of ```custom-hadoop-binary-path``` in your service-configuration is ```/pathHadoop```
2. Assume the directory path of your cluster-configuration is ``/pathConfiguration`

Reply:
Thanks, I will fix it with a new commit.